client/actions/tests/agenda_test.js

```diff
@@ -113,7 +113,7 @@ describe('agenda', () => {
                     expect(apiSpy.save.args[0]).toEqual([{}, item])
                     expect(notify.success.args[0]).toEqual(['The agenda has been created/updated.'])
 
-                    expect(dispatch.callCount).toBe(10)
+                    expect(dispatch.callCount).toBe(11)
```

Comment:
I think it is good to remove it now until we come up with a better way to test this. Will certainly look for something we can use later.